src/frontend/components/UI/PopoverComponent/index.tsx

```diff
@@ -0,0 +1,47 @@
+import Popover from '@mui/material/Popover/Popover'
+import React, { useState } from 'react'
+import './index.scss'
+
+interface PopoverComponentProps {
+  item: React.ReactElement
+  children: React.ReactElement | React.ReactNode | React.ReactNode[]
+}
+
+const PopoverComponent: React.FC<PopoverComponentProps> = ({
```

Comment:
When I open one of these popovers with a controller I'm trapped in it, I don't know how to close it. I can't move to an element outside of the popover and pressing `B` goes back in history instead of closing it. The only way to close it currently is clicking outside

Reply:
This is a bit tricky to solve, will I need to add a method on the controller side for it?

Reply:
Ok, found a way. Clicking again, or pressing the open button again should work to close it.